packages/frontend-api/src/Model/Mutation/Customer/User/CustomerUserMutation.php

```diff
@@ -68,7 +83,9 @@ public function __construct(
         CustomerUserRefreshTokenChainFacade $customerUserRefreshTokenChainFacade,
         TokenStorageInterface $tokenStorage,
         CustomerUserUpdateDataFactory $customerUserUpdateDataFactory,
-        CustomerUserFacade $customerUserFacade
+        CustomerUserFacade $customerUserFacade,
+        CustomerUserDataFactory $customerUserDataFactory,
+        TokenFacade $tokenFacade
```

Comment:
Not considered a BC break because this class is not a part of a released version, good job 👍